### PR TITLE
Add link to forum in nav

### LINKED ISSUE #5186: Add a link to the WCA forum in the navigation
Dropping this here so that it's tracked.
Requested by the WAC.

### PR DESCRIPTION
**Pull request by viroulep**
Fixes #5186.
As per WAC request.
Screenshot:
nav-forum

### CODE REVIEW

File: WcaOnRails/config/locales/es.yml [hunk 1]
```diff
@@ -1401,6 +1401,7 @@ es:
       delegates: Delegados de la WCA
       #original_hash: 37b8fdc
       organizations: Organizaciones Nacionales
+      forum: Foro
```
Comment (viroulep) [inline WcaOnRails/config/locales/es.yml]: cc @Ivan-Brigidano: same "issue", due to how our tests are written, when adding an item to the navigation we need the French and Spanish translation for it :s
Would you mind confirming or providing a suitable translation please?

Comment (Ivan-Brigidano) [reply WcaOnRails/config/locales/es.yml]: "Foro" sounds good!

Comment (viroulep) [reply WcaOnRails/config/locales/es.yml]: Thanks!

**Comment by viroulep** [comment]
Here is how it looks when added to main nav:
forum

(WAC thought it was not visible enough when nested in a menu)

**Comment by danielkcsheppard** [comment]
Awesome, approved!

**Pull request merged**